server/v2/appmanager/config.go

```diff
@@ -0,0 +1,7 @@
+package appmanager
+
+type Config struct {
+	ValidateTxGasLimit uint64 `mapstructure:"validate-tx-gas-limit"`
+	queryGasLimit      uint64 `mapstructure:"query-gas-limit"`
+	simulationGasLimit uint64 `mapstructure:"simulation-gas-limit"`
```

Comment:
Why is one public and the others private?

Reply:
need to make them all public